packages/wrangler/src/cloudchamber/create.ts

```diff
@@ -237,15 +239,19 @@ export async function handleCreateCommand(
 				return "we don't allow :latest tags";
 			}
 		},
-		defaultValue: args.image ?? "",
-		initialValue: args.image ?? "",
+		defaultValue: givenImage ?? "",
+		initialValue: givenImage ?? "",
 		helpText: 'i.e. "docker.io/org/app:1.2", :latest tags are not allowed!',
 		type: "text",
 	});
 
-	const location = await getLocation(args);
+	const location = await getLocation({
+		location: args.image ?? config.cloudchamber.location,
+	});
 	const keys = await askWhichSSHKeysDoTheyWantToAdd(args, sshKeyID);
-	const network = await getNetworkInput(args);
+	const network = await getNetworkInput({
+		ipv4: args.ipv4 ?? config.cloudchamber.ipv4,
```

Comment:
should this be `ipv4: useIpv4`?

Reply:
I initially had "use" but I switched to ipv4 since the existing CLI arg was just named "ipv4" and I figured I'd match that. Happy to change if you feel strongly one way tho!

Reply:
nah, all good